tests/functional/critical_region/src/main.c

```diff
@@ -55,7 +55,9 @@ void test_sid_pal_critical_region_with_timer(void)
  */
 extern int unity_main(void);
 
-void main(void)
+int main(void)
 {
 	(void)unity_main();
+
+	return 0;
```

Comment:
Shouldn't we return the return value from `unity_main()`

Reply:
Done